lunatrace/bsl/backend/src/models/vulnerability-dependency-tree/index.ts

```diff
@@ -55,28 +56,28 @@ export default class VulnerabilityDependencyTree {
 
     sourceManifests.forEach((manifest) => {
       manifest.child_edges_recursive?.forEach((edge) => {
-        const child = edge.child;
+        const node = edge.child;
```

Comment:
I feel like keeping this called `child` is reasonable. It is consistent with how an `edge` is defined in a tree or acyclic graph.

Reply:
actually, it's not. There is no such thing as a "child" in the database, that's simply the name of the relation. The name of the object in the database is actually "ManifestDependencyNode".. hence, node

Reply:
`childNode` might be easier to grok. By itself, I'd find `node` ambiguous.